deafrica/tests/test_chirps_non_gz.py

```diff
@@ -0,0 +1,82 @@
+import boto3
+import moto
+import pytest
+from deafrica.data.chirps import (
+    DAILY_URL_TEMPLATE,
+    MONTHLY_URL_TEMPLATE,
+    download_and_cog_chirps,
+)
+from deafrica.tests.conftest import TEST_DATA_DIR
+
+from botocore.exceptions import ClientError
+
+
+TEST_BUCKET_NAME = "test-bucket"
```

Comment:
Sorry, but can we include this set of tests in the existing test file, so there's only one file to read through for the tests. This makes for less duplication too, I think.

Reply:
I shall merge them